From: Jakob Haufe <sur5r@sur5r.net>
Date: Mon, 12 Mar 2012 00:44:57 +0100
Subject: libglbarcode uses pangocairo, not plain pango
Bug: https://sourceforge.net/tracker/?func=detail&aid=3514704&group_id=46122&atid=445116

Only having pango here results in incomplete linkage and complaints by
dpkg-shlibdeps.
---
 configure.ac                        |    2 +-
 libglbarcode/libglbarcode-3.0.pc.in |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index ebcb0c4..ca2276d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -176,7 +176,7 @@ dnl ---------------------------------------------------------------------------
 PKG_CHECK_MODULES(LIBGLBARCODE, [\
 	glib-2.0 >= $GLIB_REQUIRED \
 	cairo >= $CAIRO_REQUIRED \
-	pango >= $PANGO_REQUIRED \
+	pangocairo >= $PANGO_REQUIRED \
 ])
 
 AC_SUBST(LIBGLBARCODE_CFLAGS)
diff --git a/libglbarcode/libglbarcode-3.0.pc.in b/libglbarcode/libglbarcode-3.0.pc.in
index 325eff2..f621f14 100644
--- a/libglbarcode/libglbarcode-3.0.pc.in
+++ b/libglbarcode/libglbarcode-3.0.pc.in
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: LIBGLBARCODE
 Description: GLabels Built-In Barcode Library
-Requires: glib-2.0 cairo pango
+Requires: glib-2.0 cairo pangocairo
 Version: @VERSION@
 Libs: -L${libdir} -lglbarcode-3.0
 Libs.private: -lm
